docs/sources/alerting/old-alerting/pause-an-alert-rule.md

```diff
@@ -1,9 +1,9 @@
 +++
-title = "Pause alert rule"
+aliases = ["/docs/grafana/latest/alerting/pause-an-alert-rule/", "/docs/grafana/latest/old-alerting/pause-an-alert-rule/"]
```

Comment:
**Question:** why are we using the "old docs" for the latest alerting documentation in some cases?

Reply:
I think this is best answered by one of the technical writers in @grafana/docs-squad because I'm not certain of the relationship between these two documents.

Reply:
@achatterjee-grafana can help answer the question about 'old-alerting' docs.